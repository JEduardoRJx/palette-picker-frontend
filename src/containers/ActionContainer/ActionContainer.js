import React from 'react';
import './ActionContainer.scss';
import logo from '../../colorwheel.png'

export const ActionContainer = () => {
  return (
    <section className="action-container">
      <img src={logo} alt="Tone Zone logo, a color wheel" className="logo"/>
      <button className="randomize-btn"> Randomize </button>
      <button className="save-btn"> SAVE </button>
    </section>
  )
}

export default ActionContainer;