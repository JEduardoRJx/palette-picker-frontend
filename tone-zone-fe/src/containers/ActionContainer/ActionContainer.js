import React from 'react';

const ActionContainer = () => {
  return (
    <section>
      <p> Logo </p>
      <button className="randomize-btn"> Randomize </button>
      <button className="save-btn"> SAVE </button>
    </section>
  )
}

export default ActionContainer;