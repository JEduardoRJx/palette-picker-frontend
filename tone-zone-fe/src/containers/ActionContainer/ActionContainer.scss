@import '../../variables.scss';

.action-container {
  text-align: center;
  width: 33.3%;
}

.randomize-btn {
  background-color: #2e99a5;
  border-radius: 25px;
  box-shadow: 0 9px #999;
  color: #fff;
  cursor: pointer;
  font-size: 1.5em;
  height: 50px;
  margin: 5px;
  width: 150px;
  &:hover { 
    background-color: #11748d;
    font-style: italic;
  }
  &:active {
    background-color: #11748d;
    font-style: italic;
    box-shadow: 0 5px #666;
    transform: translateY(4px);
  }
}



.save-btn {
  background-color: #4CAF50;
  border-radius: 25px;
  box-shadow: 0 9px #999;
  color: #fff;
  cursor: pointer;
  font-size: 1.5em;
  font-weight: 700;
  height: 50px;
  margin: 5px;
  width: 150px;
  &:hover {
    background-color: #118d30;
    font-style: italic;
  }
  &:active {
    background-color: #118d30;
    font-style: italic;
    box-shadow: 0 5px #666;
    transform: translateY(4px);
  }
}

