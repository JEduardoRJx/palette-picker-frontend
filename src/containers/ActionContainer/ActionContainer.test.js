import React from 'react';
import { shallow } from 'enzyme';
import ActionContainer from './ActionContainer'

describe('ActionContainer', () => {
  it('should match the snapshot ', () => {
    const wrapper = shallow(<ActionContainer
      randomizeColors={jest.fn()}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});