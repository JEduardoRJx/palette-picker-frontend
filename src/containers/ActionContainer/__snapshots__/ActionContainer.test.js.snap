// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionContainer should match the snapshot  1`] = `
<section
  className="action-container"
>
  <img
    alt="Tone Zone logo, a color wheel"
    className="logo"
    src="colorwheel.png"
  />
  <button
    className="randomize-btn"
    onClick={[Function]}
  >
     Randomize 
  </button>
  <button
    className="save-btn"
  >
     SAVE 
  </button>
</section>
`;
